modules/cij/extractors/fio_json_read.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+"""
+Extract read metrics (iops, bwps, lat) from fio json data (fio-output* files)
+"""
+
+import os
+from typing import Collection, List, Dict
+
+from cij.runner import TestCase, trun_from_file
+from cij.extractors import fio_json
+from cij.extractors.util import dump_metrics_to_file, parse_args_load_trun
+
+_MY_NAME = os.path.splitext(os.path.basename(__file__))[0]
+
+def extract_metrics(tcase: TestCase) -> List[dict]:
+    """
+    Locate testcase fio-output files and parse them.
+    Writes metrics to aux_root/metrics.yml and returns them.
+    """
+
+    metrics = []
+
+    for fpath in fio_json.get_fio_output_files(tcase):
+        pmetric = fio_json.parse_fio_output_file(fpath)
+        if not pmetric:
+            continue
+
+        for n, job in enumerate(pmetric["jobs"]):
+            ctx = fio_json.make_context(
+                pmetric,
+                extr_name=_MY_NAME,
+                fname=os.path.basename(fpath),
+                job_id=n,
+            )
+
+            metrics.append({
+                "ctx": ctx,
+                "iops": job["read"]["iops_mean"],
+                "bwps": job["read"]["bw_mean"] * 1024,
```

Comment:
Perhaps this should be some fancy function utilizing the units?

Reply:
E.g. calling some metric-function saying "I provide **x** in unit **y** give me **x** in default-unit"

Reply:
That would be awesome. Should I give an implementation a try?

I'm thinking something like what we discussed the other day: `to_base_unit(bwps, 'KiB')` which would then convert `bwps` to bytes (since that's out base unit for storage)

Reply:
Hehe, you got here before me. Sounds like we're thinking the same thing!

Reply:
haha awesome, ``to_base_unit(bwps, 'KiB')`` sounds great!

Reply:
This should be done now

Reply:
cool